solvers/pgd.py

```diff
@@ -0,0 +1,80 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy import sparse
+    from numba import njit
+
+
+@njit
+def st_vec(x, mu):
+    return np.sign(x) * np.maximum(0., np.abs(x) - mu)
+
+
+@njit
+def prox_1d(x, l1_ratio, alpha):
```

Comment:
it's not 1d, so maybe `prox_enet` is a better name, and you can implement ST_vec inside to define one less function